tekton/cronjobs/kubectl-image-nightly-build-cron/cronjob.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: image-build-cron-trigger
```

Comment:
would it make sense to have either a README or a comment in each CronJob that explains a bit about what the cron is for?

Reply:
Yes!! I wanted to do exactly that, so I'll do it now that the conference pressure is off.

Reply:
(congrats btw!! :tada: )

Reply:
Done